mdgo/coordination.py

```diff
@@ -409,7 +409,9 @@ def heat_map(
         floating_atom: Floating atom species.
         cluster_center_sites: A list of nearest cluster center sites (atom index).
         cluster_terminal: The selection string for terminal atom species of the cluster
-            (typically the binding site for the floating ion).
+            (typically the binding site for the floating ion). The argument could be a str if
+            all the terminal atoms have the same selection string and unordered, or could be
+            a list
```

Comment:
Just some grammatical edits:

> (typically the binding site for the floating ion). The argument can be a str if
>             all the terminal atoms have the same selection string and are equivalent, or a list
> if the terminal atoms are distinct and have different selection strings.

Also, once finalized can you copy these doc changes to `get_heatmap` as well?